ts/smelter-web-client/src/smelter/offline.ts

```diff
@@ -37,24 +42,38 @@ export default class OfflineSmelter {
     });
   }
 
+  /* eslint-disable no-dupe-class-members */
```

Comment:
I feel like there should be a better way to disable that option, if not, then disable that globally